.eslintrc

```diff
@@ -6,10 +6,11 @@
   "rules": {
     "func-names": 0,
     "no-param-reassign": 0,
-    "import/prefer-default-export": 0
+    "import/prefer-default-export": 0,
+    "import/extensions": 0
   },
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 2017,
```

Comment:
Do we specifically intend to support <=ECMA2017 (as opposed to <=ECMA2016, etc?)

Reply:
There was some unresolvable linting issues that ecma 6 had introduced and the recommended solution was to move to ecma2017.

Specifically:
`myfunction(arg1, arg2, arg3,);` - where the linter says that there is a parsing error in this case because of the trailing comma but once the trailing comma is removed, says there is no trailing comma.  No other linting issues changed upon upgrading so it seems to be a cosmetic change

Reply:
Interesting... I don't care either way so sounds good.